versioned_docs/version-2.0/tutorials/observability.mdx

```diff
@@ -533,8 +537,8 @@ This will lead you back to the runs table with a filtered view:
 In this tutorial you saw how to set up your LLM application with best-in-class observability.
 No matter what stage your application is in, you will still benefit from observability.
 
-If you have more in-depth questions about observability, check out the how-to section for guides.
+If you have more in-depth questions about observability, check out the [how-to section](../how_to_guides) for guides on topics like testing, prompt management, and more.
```

Comment:
updated the topics mentioned here and below to align with our tutorials/how-to guides